include/triSYCL/detail/debug.hpp

```diff
@@ -60,63 +60,64 @@ namespace trisycl::detail {
 /** Class used to trace the construction, copy-construction,
     move-construction and destruction of classes that inherit from it
 
+    Also trace the assignments.
+
     \param T is the real type name to be used in the debug output.
 */
 template <typename T>
 struct debug {
-  // To trace the execution of the conSTRUCTORs and deSTRUCTORs
 #ifdef TRISYCL_DEBUG_STRUCTORS
   /// Trace the construction with the compiler-dependent mangled named
   debug() {
     TRISYCL_DUMP("Constructor of "
                  << boost::typeindex::type_id<T>().pretty_name()
```

Comment:
Wouldn't it be more readable to define a type alias for `boost::typeindex::type_id<T>` ? 

It is used in each TRISYCL_DUMP and maybe a shorter `type.pretty_name()` would be easier to understand at first read ?